doc/project/working_groups.rst

```diff
@@ -0,0 +1,117 @@
+TSC Working Groups
+******************
+
+Overview
+########
+
+The TSC, at its discretion, may establish working groups or subcommittees to
+serve as focused teams dedicated to specific technical areas, initiatives, or
+tasks.
+
+Membership
+##########
+
+Working Group Membership Eligibility
+++++++++++++++++++++++++++++++++++++
+
+- Each Working group (WG) shall determine its own membership eligibility, in
+  consultation with the TSC.
+- Each working group shall have a core team of members who are actively involved
+  in its activities and decision-making processes.
+- It is expected that WG core membership shall be **open to all Zephyr project
+  Contributors**; however, working groups may impose restrictions such as the
+  number of participants from a single company.
```

Comment:
Not sure this is needed and I would actually reword things here. Reason being is if a company happens to have a lot of valuable individuals that can contribute to the project.

The IEEE charter/ethics tries to address this by with a general line:
 
- "to avoid real or perceived conflicts of interest whenever possible...". 
- "to seek, accept, and offer honest criticism of technical work..."

The jist of this is that IEEE members are supposed to represent as individuals and value all technical contributions and not drive company interest at the detriment of the openness of the IEEE WG. How this is generally managed is by the higher level management, in our case the TSC, to field and review cases where a single company is not behaving in an open and clear manner. IEEE has shut down WG where a company-who-shall-not-be-named was not operating in a proper manner.